pkg/hook/hook.go

```diff
@@ -110,6 +117,7 @@ func (h *Hook) Run(bindingType BindingType, context []BindingContext, logLabels
 		envs = append(envs, fmt.Sprintf("BINDING_CONTEXT_PATH=%s", contextPath))
 		envs = append(envs, fmt.Sprintf("METRICS_PATH=%s", metricsPath))
 		envs = append(envs, fmt.Sprintf("VALIDATING_RESPONSE_PATH=%s", validatingPath))
+		envs = append(envs, fmt.Sprintf("KUBERNETES_OBJECT_PATCH_SET_PATH=%s", objectPatchPath))
```

Comment:
I'd rather name it "KUBERNETES_PATCH_PATH".

Reply:
Renamed!